posthog/api/test/test_feature_flag.py

```diff
@@ -152,6 +155,11 @@ def test_create_feature_flag(self, mock_capture):
             },
         )
 
+        historical_flags = HistoricalVersion.objects.filter(team_id=self.team.id, name="FeatureFlag")
```

Comment:
To reduce dependencies on internal details, and make it easier to refactor, can we use the history http api instead of querying the db directly?